src/dos.c

```diff
@@ -1803,7 +1804,7 @@ void intr21()
     {
         // Note: ignore drive letter in DL
         const uint8_t *path = dos_get_cwd(cpuGetDX() & 0xFF);
-        debug(debug_dos, "\tcwd '%c' = '%s'\n", '@' + (cpuGetDX() & 0xFF), path);
+        debug(debug_dos, "\tcwd '%c' = '%s'\n", (char)('@' + (cpuGetDX() & 0xFF)), path);
```

Comment:
This is strange - as a '%c' format expects an integer argument - see C23, 7.23.6.1 p8.